test/unitary/session/streams/SeqNum.cpp

```diff
@@ -0,0 +1,81 @@
+#include <gtest/gtest.h>
+
+extern "C"
+{
+#include <c/core/session/stream/seq_num.c>
+}
+
+TEST(SeqNumTest, AddFromNegative)
+{
+    EXPECT_EQ(0, uxr_seq_num_add(uxrSeqNum(-1), 1));
```

Comment:
This is the eternal war between the `~0` and the `-1`. 
Yes, if we follow the same rationale that we talk, it could be platform dependent. I think that in this case, the `~0u` is the safe way.